.gitlab/build_corona.yml

```diff
@@ -19,7 +19,7 @@
 .src_build_on_corona:
   stage: build
   variables:
-    ALLOC_COMMAND: "srun -p pdebug -t 25 -N 1 "
+    ALLOC_COMMAND: "srun -p pbatch -t 25 -N 1 "
```

Comment:
I'm confused -- is `pbatch` not available? If not, should we hold off on this PR until it is?

Reply:
I'm confused as well. We should be running CI in pbatch.

Reply:
Mistake #1: assuming there is anything useful in any of Rich's messages....